.github/workflows/build.yml

```
@@ -47,26 +47,25 @@ jobs:
       run: poetry install
 
     - name: Create config file
-      run: make config-file
+      run: makim config.file
 
-    - name: start up docker container
+    - name: start up containers-sugar
       run: |
-        make docker-compose-build
-        make docker-compose-start
-        make docker-compose-wait SERVICE=postgres
-
+        containers-sugar build
+        containers-sugar start
+        #make docker-compose-wait SERVICE=postgres
```

```suggestion
        makim containers.wait --service=postgres
```
you would need to implement on the makim config file